eventmesh-admin-server/build.gradle

```diff
@@ -31,18 +31,25 @@ dependencies {
     implementation "io.grpc:grpc-stub"
     implementation "io.grpc:grpc-netty-shaded"
 
-    // https://mvnrepository.com/artifact/com.baomidou/mybatis-plus-boot-starter
-    implementation 'com.baomidou:mybatis-plus-boot-starter:3.5.7'
-    implementation "org.reflections:reflections:0.10.2"
+	// https://mvnrepository.com/artifact/com.baomidou/mybatis-plus-boot-starter
+	implementation "com.baomidou:mybatis-plus-boot-starter"
 
-    // https://mvnrepository.com/artifact/com.alibaba/druid-spring-boot-starter
-    implementation "com.alibaba:druid-spring-boot-starter"
-    compileOnly 'com.mysql:mysql-connector-j'
-    compileOnly 'org.projectlombok:lombok'
-    annotationProcessor 'org.projectlombok:lombok'
+	// https://mvnrepository.com/artifact/com.alibaba/druid-spring-boot-starter
+	implementation "com.alibaba:druid-spring-boot-starter"
+	implementation 'com.mysql:mysql-connector-j'
+	compileOnly 'org.projectlombok:lombok'
```

Comment:
`mysql-connector-j` artifact with `implementation` namespace introduces incompatible license.

I will submit a PR and add `The GNU General Public License, v2 with Universal FOSS Exception, v1.0` in `deniedLicenses` later.

Reply:
Please see https://github.com/apache/eventmesh/pull/5043